Cargo.toml

```diff
@@ -9,15 +9,16 @@ license = "MIT"
 name = "nom_locate"
```

Comment:
I think you deserve to add yourself as an author of the crate ;). Thanks for your contributions !